newton/_src/sim/builder.py

```
@@ -1394,68 +1393,152 @@ def add_mjcf(
 
     # endregion
 
+    # region World management methods
+
+    def begin_world(self, key: str | None = None, attributes: dict[str, Any] | None = None):
```

Ruff (0.14.7)

Unused method argument: `key`

(ARG002)